napari/_qt/dialogs/activity_dialog.py

```diff
@@ -0,0 +1,68 @@
+from qtpy.QtCore import QSize, Signal
+from qtpy.QtWidgets import (
+    QDialog,
+    QFrame,
+    QGraphicsOpacityEffect,
+    QHBoxLayout,
+    QLabel,
+    QScrollArea,
+    QSizePolicy,
+    QVBoxLayout,
+    QWidget,
+)
+
+MIN_WIDTH = 250
+MIN_HEIGHT = 140
+
+
+class ActivityDialog(QDialog):
+    """Activity Dialog for Napari progress bars."""
+
+    resized = Signal(QSize)
+    closed = Signal()
+
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.setObjectName('Activity')
+        self.setMinimumWidth(MIN_WIDTH)
+        self.setMinimumHeight(MIN_HEIGHT)
+        self.setMaximumHeight(MIN_HEIGHT)
+        self.setSizePolicy(
+            QSizePolicy.MinimumExpanding, QSizePolicy.MinimumExpanding
+        )
+
+        opacity_effect = QGraphicsOpacityEffect(self)
+        opacity_effect.setOpacity(0.8)
+        self.setGraphicsEffect(opacity_effect)
+
+        self.baseWidget = QWidget()
+
+        self.activity_layout = QVBoxLayout()
+        self.activity_layout.addStretch()
+        self.baseWidget.setLayout(self.activity_layout)
+
+        self.scroll = QScrollArea()
+        self.scroll.setWidgetResizable(True)
+        self.scroll.setWidget(self.baseWidget)
+
+        self.title_bar = QLabel()
+
+        title = QLabel('activity', self)
```

Comment:
Maybe `Activity` uppercased?

Reply:
Also it should be translatable, so 

`trans._('Activity')`

Reply:
Yeah I wasn't sure about uppercasing or not. The reason I went with lowercase is because that's how we've commonly done it for dock widgets e.g. 'layer controls' and 'console'. Happy to change to uppercase if we want.

Reply:
Yeah we don't have a uniform policy... A remnant of @freeman-lab's (:wave:) and my dislike of uppercase in general. :joy: I must reluctantly admit that most UI elements in most software are capitalized and we should probably follow that convention.

```suggestion
        title = QLabel(trans._('Activity'), self)
```